pybamm/spatial_methods/finite_volume.py

```diff
@@ -146,11 +146,25 @@ def divergence(self, symbol, discretised_symbol, boundary_conditions):
                 out = divergence_matrix @ ((r_edges**2) * discretised_symbol)
             elif submesh.coord_sys == "cylindrical polar":
                 out = divergence_matrix @ (r_edges * discretised_symbol)
-        else:
+        elif submesh.coord_sys == "cartesian":
             out = divergence_matrix @ discretised_symbol
-
+        else:
+            raise ValueError(
+                f"Coordinate system is {submesh.coord_sys}, not in cartesian, cylindrical polar, spherical polar")
         return out
 
+    def check_coordinate_system(self, symbol):
+        """
+        Double check coordinate system
+        """
+        submesh = self.mesh[symbol.domain]
+        if submesh.coord_sys not in ["cylindrical polar", "spherical polar","cartesian"]:
```

Comment:
This line is too long and it crashes the precommit, please reduce so it is maximum 88 characters.